fvm/Equation/AdvectionInterpolationCoefficient.cpp

```diff
@@ -14,11 +14,13 @@ using namespace std;
  * Constructor.
  */
 AdvectionInterpolationCoefficient::AdvectionInterpolationCoefficient(Grid*g, fluxGridType fluxGridType, 
-        adv_bc bc_l, adv_bc bc_u){
+        adv_bc bc_l, adv_bc bc_u, len_t offset){
```

Comment:
Could you add a comment above describing the various input parameters? Something like:

```
g:            Grid on which the advection term lives.
fluxGridType: Coordinate in which we will interpolate (radius, p1 or p2).
bc_l:         Boundary condition to assume on the lower boundary.
bc_u:         Boundary condition to assume on the upper boundary.
offset:       For quantities with multiples, the offset into the quantity for
              which interpolation should be done (e.g. ion charge state
              index).
```